src/allotropy/parsers/cytiva_unicorn/reader/strict_element.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
```

Comment:
This seems generally useful for reading XML files, can you move this into `parser/utils/strict_xml_element.py`